R/mainTests.r

```diff
@@ -967,29 +979,18 @@ overcompleted_cols <- function(data, meta) {
 
   overcomplete_regional_cols <- function(matrixRow) {
     # Start by filtering the data down to remove the geographic level being tested and any lower levels we don't care about
```

Comment:
Should we update or remove this comment now that we're explicitly stating the levels we care about? Maybe change to 'these are the levels that we would expect the region columns to always be empty for?'

Reply:
Done